index.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿